tasks/install_targz.yml

```diff
@@ -50,15 +52,41 @@
     src: app@.service.j2
     dest: "{{ instance_systemd_path }}{{ instance_systemd_unit_file }}"
   any_errors_fatal: true
-
-- name: Copy an unit-file to /etc/systemd/system/
-  become: true
-  copy:
-    remote_src: true
-    src: "{{ instance_systemd_path }}{{ instance_systemd_unit_file }}"
-    dest: "/etc/systemd/system/{{ instance_systemd_unit_file }}"
-    owner: root
-    group: root
-  any_errors_fatal: true
   notify:
     - reload-systemd-daemon
+
+- name: Configure stateboard
+  block:
+    - name: Set stateboard facts
+      set_fact:
+        stateboard_name: '{{ cartridge_app_name }}-stateboard'
+        stateboard_work_dir: '{{ cartridge_app_name | get_instance_work_dir(inventory_hostname, True, data_dir) }}'
+        stateboard_run_dir: '{{ cartridge_app_name | get_instance_run_dir(inventory_hostname, True, run_dir) }}'
+        stateboard_conf_file: '{{ cartridge_app_name | get_instance_conf_file(inventory_hostname, True, conf_dir) }}'
+
+    - name: Create stateboard folders
+      file:
+        path: "{{ item.path }}"
+        state: directory
+        mode: "{{ item.mode | default('0755') }}"
+        owner: "{{ instance_user }}"
+        group: "{{ instance_group }}"
+      loop:
+        - {path: "{{ stateboard_work_dir }}"}
+        - {path: "{{ stateboard_run_dir }}"}
+      any_errors_fatal: true
+
+    - name: Create a stateboard service file
+      template:
+        src: stateboard.service.j2
+        dest: "{{ instance_systemd_path }}{{ stateboard_name }}.service"
+      any_errors_fatal: true
+      notify:
+        - reload-systemd-daemon
+  tags: always
+  when: stateboard_on_machine == true
+
+-  name: trigger reload-systemd-daemon
+   command: /bin/true
```

Comment:
It can be triggered only if previous tasks changed something.
Triggers are flushed once after package installing, so it's OK to add notify for reach task that changes files (in fact, we need to reload systemd daemon only if unit files were changed).

Reply:
handlers are only triggered when a change happens. For some reason, I came across cases, when changing or creating a service file did not set "change" state. I try to reproduce it now...

Reply:
Found it: changing service file does not trigger instance restart. This may be required when switching between `ansible-cartridge` version.

But I see what you were trying to do, so I will remove this functionality. And create an issue